public/docs/ts/latest/tutorial/toh-pt2.jade

```diff
@@ -105,206 +110,214 @@ code-example(language="bash").
     and stamps out instances of this template.
 
     The quoted text assigned to `ngFor` means
-    “*take each hero in the `heroes` array, store it in the local `hero` variable,
-    and make it available to the corresponding template instance*”.
+    “*in the `heroes` array, store each hero in the local `hero` variable
+    and make it available to the corresponding template instance*.”
 
     The `let` keyword before "hero" identifies `hero` as a template input variable.
-    We can reference this variable within the template to access a hero’s properties.
+    We can reference this variable within the template to access a hero's properties.
 
     Learn more about `ngFor` and template input variables in the
     [Displaying Data](../guide/displaying-data.html#ngFor) and
-    [Template Syntax](../guide/template-syntax.html#ngFor) chapters.
+    [Template Syntax](../guide/template-syntax.html#ngFor) pages.
 
 :marked
-  Now we insert some content between the `<li>` tags
-  that uses the `hero` template variable to display the hero’s properties.
+  Next, update the `<li>` tags to add content
+  that uses the `hero` template variable to display the hero's properties.
 
 +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'ng-for', 'app.component.ts (ngFor template)')(format=".")
 
 :marked
-  When the browser refreshes, we see a list of heroes!
+  When the browser refreshes, you see a list of heroes.
 
   ### Styling our heroes
-  Our list of heroes looks pretty bland.
-  We want to make it visually obvious to a user which hero we are hovering over and which hero is selected.
+  The list of heroes looks pretty bland.
+  We want to give users a visual cue of which hero they are hovering over and which hero is selected.
 
-  Let’s add some styles to our component by setting the `styles` property on the `@Component` decorator
+  To add some styles to your component, set the `styles` property on the `@Component` decorator
   to the following CSS classes:
 
 +makeExample('toh-2/ts/app/app.component.ts', 'styles', 'app.component.ts (styles)')(format=".")
 
 :marked
-  Notice that we again use the back-tick notation for multi-line strings.
+  Notice that again you use the backtick notation for multi-line strings.
 
-  That's a lot of styles! We can put them inline as shown here, or we can move them out to their own file which will make it easier to code our component. 
-  We'll do this in a later chapter. For now let's keep rolling.
+  That's a lot of styles! In a later page, you'll move the styles to a separate file. 
```

Comment:
"That's a lot of styles!" is a conversational style we've been working to reduce. I'd take it out and reword the sentence immediately afterward to include that idea.